src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Parser/SbomFileParser.cs

```diff
@@ -100,25 +96,25 @@ public long GetSbomFile(ref byte[] buffer, ref Utf8JsonReader reader, out SBOMFi
         }
     }
 
-    private void ValidateSbomFile(SBOMFile sbomFile)
+    private void ValidateSbomFile(SPDXFile sbomFile)
```

Comment:
Not terribly important, but does it make sense to call this `ValidateSpdxFile` given the parameter type name change?

Reply:
I will update this in the final PR, will take it as a note.